component/site/libraries/iCalImport.php

```diff
@@ -707,24 +707,30 @@ public static function convertWindowsTzid($wtzid){
 		$wtzdata["Fiji, Kamchatka, Marshall Is."] = "Etc/GMT-12";
 		$wtzdata["Chatham Islands"] = "Pacific/Chatham";
 		$wtzdata["Nuku'alofa"] = "Pacific/Tongatapu";
-		$wtzdata["Kiritimati"] = "Pacific/Kiritimati";		
+		$wtzdata["Kiritimati"] = "Pacific/Kiritimati";
 		$wtzdata["Central Standard Time"] = "America/Chicago";
 
 		// manual entries
 		$wtzdata["GMT -0500 (Standard) / GMT -0400 (Daylight)"] = "America/New_York";
-		$wtzdata["Eastern Standard Time"] = "America/New_York";		
+		$wtzdata["Eastern Standard Time"] = "America/New_York";
 		$wtzdata["W. Europe Standard Time"] = "Europe/Paris";
 		$wtzdata["E. Europe Standard Time"] = "Europe/Helsinki";
 		$wtzdata["FLE Standard Time"] = "Europe/Helsinki";
 		$wtzdata["Mountain Standard Time"] = "America/Denver";
 		$wtzdata["Romance Standard Time"] = "Europe/Brussels";
 		$wtzdata["GMT Standard Time"] = "UTC";
 		$wtzdata["Tasmania Standard Time"] = "Australia/Hobart";
-		
+
+		// Lets check if a file for custom timezones exists
+		if (JFile::exists(JPATH_COMPONENT_SITE . '/libraries/ical_custom_timezones.php')) {
```

Comment:
I know, but it doesn't look to be there? I've just been using my laptop and merged the upstream jQuery branch.